PACKAGES.md

```diff
@@ -0,0 +1,27 @@
+### Most Core Packages & Examples
+| Name          | Description           
+| ------------- |:-------------:
+| [prelude](https://github.com/mostjs/prelude) | Base functional programming utils for mostjs packages
+| [stream](https://github.com/mostjs/stream)     | Core stream type  
```

Comment:
Let's remove this one for now.  We're not using it yet, so I'd rather not promote it.